Dockerfile

```diff
@@ -9,7 +9,9 @@ RUN apt-get update && \
     apt-get install -y -q build-essential \
     python3-dev  \
     libpq-dev \
-    curl
+    curl \
+    librdkafka-dev \
+    build-essential
```

Comment:
`build-essential` is already on 9th line

Reply:
good point. fixed